ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-stable-4.12-upgrade-from-stable-4.11.yaml

```diff
@@ -107,25 +107,6 @@ tests:
     test:
     - chain: openshift-upgrade-qe-sanity
     workflow: cucushift-installer-rehearse-azure-ipi-fullyprivate-proxy
-- as: gcp-ipi-disc-priv-oidc-fips-f60
-  cron: 17 14 25 2,4,6,8,10,12 *
-  steps:
-    cluster_profile: gcp-qe
-    env:
-      ENABLE_OTA_TEST: OCP-30832
-      FIPS_ENABLED: "true"
-    test:
-    - chain: openshift-upgrade-qe-sanity-disconnected
-    workflow: cucushift-installer-rehearse-gcp-ipi-disconnected-private-cco-manual-workload-identity
-- as: gcp-ipi-proxy-private-f60
-  cron: 38 12 23 2,4,6,8,10,12 *
-  steps:
-    cluster_profile: gcp-qe
-    env:
-      ENABLE_OTA_TEST: OCP-24358
```

Comment:
@dis016 and this one

Reply:
`OCP-24358` is also enabled on other platforms.